packages/perseus-editor/src/item-editor.tsx

```diff
@@ -210,11 +210,6 @@ class ItemEditor extends React.Component<Props, State> {
                                         url={this.props.previewURL}
                                     />
                                 </DeviceFramer>
-                                <div
-                                    id="hintsarea"
-                                    className="hintsarea"
-                                    style={{display: "none"}}
-                                />
```

Comment:
This `<div>` is completely unused and I couldn't find any selectors that use it.